qlib/workflow/online/utils.py

```diff
@@ -127,45 +128,74 @@ def get_online_tag(self, recorder: Recorder) -> str:
         tags = recorder.list_tags()
         return tags.get(self.ONLINE_KEY, self.OFFLINE_TAG)
 
-    def reset_online_tag(self, recorder: Union[Recorder, List]):
+    def reset_online_tag(self, recorder: Union[Recorder, List], exp_name: str = None):
         """
         Offline all models and set the recorders to 'online'.
 
         Args:
             recorder (Union[Recorder, List]):
                 the recorder you want to reset to 'online'.
+            exp_name (str): the experiment name. If None, then use default_exp_name.
 
         """
+        if exp_name is None:
```

Comment:
This piece of code appears too many times.
Please create a new function like  `def _get_exp_name(exp_name) -> str`  (error may be raised in this method)